Organic/PageInjection.php

```diff
@@ -415,8 +415,7 @@ function t() {
                 <?php
                 if (
                         $this->organic->getOrganicPixelTestValue() &&
-                        $this->organic->getOrganicPixelTestPercent() != '' &&
-                        $this->organic->getOrganicPixelTestPercent() != null
+                        $this->organic->getOrganicPixelTestPercent() !== null
```

Comment:
Can we at least check if it's a number? It will likely mess up the `enabled: <value>` Javascript if not

Reply:
`organicPixelTestPercent` is defined using `intval` function which will return 0 for failure and int value in case of successful string->int conversion.
https://github.com/orgnc/wordpress-plugin/blob/4f02059389460bf33882a093135cc1948d0be920/Organic/Organic.php#L263